app/src/NativeBase/Containers/ProfileContainer.tsx

```diff
@@ -13,20 +44,72 @@ import React from 'react'
 const ProfileContainer = () => {
   return (
     <VStack
-      paddingTop="4"
-      paddingBottom="2"
-      alignItems="center"
-      space="4"
-      backgroundColor="secondaryGrey.60"
+      padding={2}
+      backgroundColor="bg.100"
       _dark={{ backgroundColor: 'bgDarkMode.100' }}
     >
-      <Center
-        backgroundColor="bg.100"
-        _dark={{ backgroundColor: 'bgDarkMode.100' }}
-        width="100%"
-      >
-        <Heading size="sm">My Profile</Heading>
+      <Center>
+        <Heading fontFamily="title" size="lg">
+          My Profile
+        </Heading>
+        <Box
+          marginTop={6}
+          height="93px"
+          width="90px"
+          borderColor="darkGrey.100"
+          borderWidth="4px"
+          borderRadius="full"
+        />
+        <HStack>
+          <Heading fontSize="md" fontWeight="700">
+            Full Name
+          </Heading>
+          <Text fontSize="xs" paddingLeft="-2">
```

Comment:
In this screen a mix of sizes is used. The size `xs` is used 4x, `sm` 3x, `md` 2x. 

The `Text` component in `StaTheme` has default size 'md'. If I add fontSize="xs" to the "HStack" and delete the xs props then the font goes bigger overall.  Not sure where else to add a default prop in the theme?

Reply:
Two thoughts about possible ways could do this:

- Create some custom semantically named custom font sizes in StaTheme under `fonts.sizes` (e.g. `formLabel: 16,` -- that's just an example name, use whatever names makes most sense)  Then you could reference them here as e.g. `<Text fontSize="formLabel"`.  That way if we decide to change the size of e.g. form labels across the app, we could change it one place rather than dozens.

- Or could create reusable React components with the font sizes set there, but that might get a bit tricky if there are lots of different padding and margin settings going on

Reply:
(And yep, no worries not to set it on the `HStack` if that doesn't help)

Reply:
Actually, ignore my first bullet point above -- I think this would be anti-pattern (it would work, but NativeBase docs don't suggest anything like this -- they stick to standard font size names like `xs`)

I don't feel super comfortable with all the different font sizes and paddings/margins being set on each element -- we should have a design system where we use reusable components (with common settings around sizing and spacing) in most places in the app, so that creating new screens (and updating any settings later) is much easier and more standardised.  

I don't know how easy it is to standardise things with the current designs (e.g. I don't know if Mia is working that way in Figma), so I'll leave it up to you what you think is possible / makes sense here.

Reply:
I agree Joe and feel the same way. Last week had a chat with Mia and she uses the descriptions that are in the Typography section of the Design System so we can see how to make that easier to use in our theme. The same could apply to the padding and margins. For this screen I can't see an easy way not to use so many margins on the fly, but we can have a deep dive into making that better as part of the ticket that refactors the `Text` element.

Reply:
Thanks for looking into it Pam -- that makes sense 👍